addon/lib/toolbar-button.js

```diff
@@ -68,9 +72,18 @@ function getExperimentList(availableExperiments, installedAddons) {
     }
     return experiment;
   })
-  .filter(x => x.active || !x.completed); // remove inactive, completed experiments
-
-  // Sort new experiments to the top, otherwise sort by reverse-chronological
+  // Remove inactive, completed experiments.
+  .filter(x => x.active || !x.completed)
+  // Remove experiments with a locale_grantlist, unless the user's primary
+  // locale is in the grantlist. #1961
+  .filter(x => !('locale_grantlist' in x) || x.locale_grantlist.includes(userLocale))
```

Comment:
In python I would prefer to write this x.get('locale_grantlist', []).includes(userLocale) but I'm not sure how to write that in js.